src/lib/Form/Type/ChoiceList/Loader/LanguageChoiceLoader.php

```diff
@@ -8,99 +8,22 @@
 
 namespace EzSystems\EzPlatformAdminUi\Form\Type\ChoiceList\Loader;
 
-use eZ\Publish\API\Repository\LanguageService;
-use Symfony\Component\Form\ChoiceList\ArrayChoiceList;
-use Symfony\Component\Form\ChoiceList\Loader\ChoiceLoaderInterface;
-
-class LanguageChoiceLoader implements ChoiceLoaderInterface
+class LanguageChoiceLoader extends ConfiguredLanguagesChoiceLoader
```

Comment:
I'm not so sure about this one for `1.5`. It's a bit of a BC break - a change of a behavior. This loader is used for e.g. Content Create widget. So far it was possible to create Content in any system language, regardless of if it's present in `admin_group` prioritized languages list. That lead to other issues, true, but still feels a bit too much for `1.5`.

The key difference in the behavior is `array_merge($saLanguages, array_values($languagesByCode));` in `getChoiceList` method - it returns a merged list of SA languages and any other system languages. With this change, if I got it right, it would return SA languages only.

POV ping @webhdx

Reply:
This is a mistake, it should be backward compatibility. I've updated the logic in ConfiguredLanguagesChoiceLoader.